opencti-platform/opencti-graphql/src/migrations/1691569743543-add-entity-setting-workflow_configuration.js

```diff
@@ -0,0 +1,22 @@
+import { elUpdateByQueryForMigration } from '../database/engine';
+import { READ_INDEX_INTERNAL_OBJECTS } from '../database/utils';
+import { ENTITY_TYPE_ENTITY_SETTING } from '../modules/entitySetting/entitySetting-types';
+
+export const up = async (next) => {
```

Comment:
One last point, you can add logApp.info 'started' and 'stoped' to be consistent with others migrations

Reply:
it's included in the elUpdateByQueryForMigration function